kolibri/plugins/user_auth/kolibri_plugin.py

```diff
@@ -55,3 +55,8 @@ def url(self):
 @register_hook
 class LogInNavAction(NavigationHook):
     bundle_id = "login_side_nav"
+
+
+@register_hook
+class LogOutNavAction(NavigationHook):
```

Comment:
See `kolibri/core/kolibri_plugin.py` for where this should be (and currently is) defined.